buildSrc/src/main/kotlin/common-conventions.gradle.kts

```
@@ -1,5 +1,2 @@
 val deployVersion = project.findProperty("DeployVersion")
```

```suggestion
val deployVersion: String? = project.findProperty("DeployVersion")?.toString()?.ifBlank { null }
```

It's unrelated to this PR, but I'd recommend being defensive here, just in case a plain-string gets passed.

I'll leave it up to you whether to implement or not, but I wanted to mention it while it's on my mind.

---

Thanks, applied